tests/testthat/test-across.R

```diff
@@ -888,6 +888,32 @@ test_that("if_any() and if_all() expansions deal with no inputs or single inputs
   )
 })
 
+test_that("if_any() on zero-column selection behaves as expected in filter", {
```

Comment:
```suggestion
test_that("if_any() on zero-column selection behaves like any() (#7059)", {
```

We often like to link to the issue when we add a test specifically for it

Reply:
No problem - will update that!